modules/arch-platform-operators.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+// * operators/admin/olm-managing-po.adoc
+
+:_content-type: CONCEPT
+
+ifeval::["{context}" == "control-plane"]
+[id="platform-operators_{context}"]
+= Platform Operators (Technology Preview)
+
+:FeatureName: The platform Operator type
+include::snippets/technology-preview.adoc[]
+endif::[]
+
+ifeval::["{context}" == "olm-managing-po"]
+[id="platform-operators_{context}"]
+= About platform Operators
+endif::[]
+
+Operator Lifecycle Manager (OLM) introduces a new type of Operator called _platform Operators_. A platform Operator is an OLM-based Operator that can be installed during or after an {product-title} cluster's Day 0 operations and participates in the cluster's lifecycle. As a cluster administrator, you can use platform Operators to further customize your {product-title} installation to meet your requirements and use cases.
+
+Using the existing cluster capabilities feature in {product-title}, cluster administrators can already disable a subset of Cluster Version Operator-based (CVO) components considered non-essential to the initial payload prior to cluster installation. Platform Operators iterate on this "composable OpenShift" model by providing additional customization options. Through the platform Operator mechanism, OLM-based Operators can now be installed at cluster installation time and can block cluster rollout if the Operator fails to install successfully.
+
+In {product-title} 4.12, this Technology Preview release focuses on the basic platform Operator mechanism and builds a foundation for expanding the concept in upcoming releases. You can use the cluster-wide `PlatformOperator` API to configure Operators before or after cluster creation on clusters that have enabled the `TechPreviewNoUpgrades` feature set.
```

Comment:
This is probably due to my limited understanding but the second sentence is a little confusing - how can you have the tech preview no upgrades feature set enabled on clusters before they are created? (Just making sure "on clusters that have enabled the `TechPreviewNoUpgrades`" part is correct here - ignore if it is)

Reply:
Planning on having a follow-up PR that adds the "before cluster install" procedure. TLDR: you add a `FeatureGate` and `PlatformOperators` object to a `manifests/` folder before creation and point the installer there.